File: remoteTrainer/app/src/main/java/zavrsni/besednik/com/remotetrainer/model/models/Exercise.java
package zavrsni.besednik.com.remotetrainer.model.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Exercise implements Serializable {
    @SerializedName("IDExercise")
    private int id;
    @SerializedName("Name")
    private String name;
    @SerializedName("Description")
    private String description;
    @SerializedName("Video")
    private Video video;

    public int getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public String getDescription() {
        return description;
    }

    public Video getVideo() {
        return video;
    }

    public class Video implements Serializable {
        @SerializedName("Name")
        private String name;
        @SerializedName("Path")
        private String path;

        public String getName() {
            return name;
        }

        public String getPath() {
            return path;
        }
    }
}

File: web servis/RemoteTrainerAPI/Models/ResponseModels/Classes/Exercise.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace RemoteTrainerAPI.Models.ResponseModels.Classes
{
    public class Exercise
    {
        public int IDExercise { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
    }
}